docs/shared-content-source/docs/modules/ROOT/partials/include.adoc

```diff
@@ -1,4 +1,35 @@
-// global attributes
-:page-partial:
 
-:cli-plugin: kubectl
+// global attributes if you update, make sure to update in docs/modules/RoOT:partials.adoc too
+
+:supported-java-v: >= 8
+:supported-sbt-v: version 1.2.8 or higher
+:supported-docker-v: version 18.09 or higher
+:supported-helm-v: version 3 or higher
+
+:kubernetes-platform-name:     Kubernetes
+
+:kubectl-plugin-version:       KUBECTL_PLUGIN_VERSION
+
+:cli:                          kubectl
+:cli-plugin:                   kubectl
+
+:latest-openshift-platform-version:        3.11
+:minimum-kubernetes-on-openshift-version:  1.11
+:minimum-openshift-platform-version:       3.11
+:minimum-platform-version-major-dot-minor: 3.11
+:minimum-sbt-version:                      0.13.7
+:latest-sbt-version: 					   1.3.13
+
+
+:supported-akka-v: 2.6.6 
```

Comment:
not introduced in this PR, but this is outdated ... cc. @RayRoestenburg 
we need to find a "sane" way of keeping those numbers updated ...